ykrt/src/trace/swt/patch.rs

```diff
@@ -0,0 +1,134 @@
+use libc::{mprotect, size_t, sysconf, PROT_EXEC, PROT_READ, PROT_WRITE};
```

Comment:
Let's add a docstring at the top of the file explaining what the high-level intent is. And also, probably, a big fat warning that it's dangerous, but improves performance.

Reply:
added 👉 0bd8f3617e95057590fafe45a72eecf6cd96a922